src/Parameters.jl

```diff
@@ -0,0 +1,23 @@
+module Parameters
+
+abstract type AbstractInsolationParams end
+const APS = AbstractInsolationParams
+
+Base.@kwdef struct InsolationParameters{FT} <: AbstractInsolationParams
+    obliq_epoch::FT
```

Comment:
This is just being picky, but can we reorder this list? Just to have some of the more generic planet parameters all first and then all the orbital parameters which are time dependent and depend on the epoch after.

I would have it go: `year_anom`, `day`, `orbit_semimaj`, `tot_solar_irrad`, `epoch`, `mean_anom_epoch`, `eccentricity_epoch`, `obliq_epoch`, `lon_perihelion_epoch`.

Reply:
Yes, absolutely, done!